config/20.1/make.conf

```
@@ -63,7 +63,7 @@ net_freeradius3_UNSET=		HEIMDAL
 net_frr6_SET=			SNMP
 net_frr7_SET=			SNMP
 net_haproxy_SET=		LUA
-net_miniupnpd_SET=		CHECK_PORTINUSE PF_FILTER_RULES
+net_miniupnpd_SET=		PF_FILTER_RULES
```

```suggestion
net_miniupnpd_SET=		CHECK_PORTINUSE  PF_FILTER_RULES
```

Here this is more complicated, it's broken in FreeBSD somewhat, 12 ok, 11 not ok: https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=233963

we just flipped the logic but ideally FreeBSD must be fixed to enable both to work again

sorry to make this so complicated, but we need the most flexible fix so that it doesn't break left and right and enables us to not worry about it anymore

---

Oh crap, that's so screwed up :-(
But at least there is some progress.

I completely agree to stay as near as possible to the FreeBSD source (and the options),
but for now, it's just not possible to build with that bug, and there is a working workaround.

For now i think that this is the only one, which completely breaks the build procedure, so why not temporary apply that workaround, until its fixed by FreeBSD?

---

because there's nothing more persistent than a temporary workaround

---

(I asked for commit in the freebsd ticket, the patch was ready but forgotten it seems)

---

So well, i definitely have to agree with that ^^

So lets keep everything in place, and hope that they will commit it asap.

---

Almost half a year later. Need to ping FreeBSD to apply the patch soon.